client/my-sites/site-settings/speed-up-site-settings.jsx

```diff
@@ -102,7 +108,12 @@ class SpeedUpSiteSettings extends Component {
 								text={ translate(
 									"Delays the loading of images until they are visible in the visitor's browser."
 								) }
-								link="https://jetpack.com/support/lazy-images/"
+								link={
+									siteIsAtomic
+										? 'https://wordpress.com/support/settings/performance-settings/#lazy-load-images'
+										: 'https://jetpack.com/support/lazy-images/'
+								}
+								privacyLink={ ! siteIsAtomic }
```

Comment:
Privacy links [shouldn't be touched](https://github.com/Automattic/wp-calypso/issues/59976#:~:text=the%20links%20for%20Privacy%20Information%20can%20stay%20as%20is). To keep things as they were, I think this should be:
```suggestion
							privacyLink="https://jetpack.com/support/lazy-images/#privacy"
```